### ISSUE TITLE
[BUG] Update 1.11.1 Fails to Launch on Windows 7

### ISSUE THREAD

**Issue by zellchristensen** [oxen-io/session-desktop#2898]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-desktop/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After the most recent automatic update, the client fails to launch on Windows 7 with the error:

> The procedure entry point DiscardVirtualMemory could not be
located in the dynamic link library KERNEL32.dll.

![2023-09-04_02-47-09](https://github.com/oxen-io/session-desktop/assets/38737447/6d1d2471-341a-4702-b783-d60e169fe08e)

### Expected Behavior

The client launches.

### Steps To Reproduce

In Windows 7, Start Session

### Desktop Version

v1.11.1

### Anything else?

I'm assuming this is related to Electron dropping support for Windows 7/8/8.1

If Windows 7 is no longer supported, there should be clear language on the website, github, and elsewhere clarifying which platforms are supported.

**Comment by Jigsy1** [comment]
On Windows 8.1 and had the same problem. Ended up rolling back to the previous version.

**Comment by KeeJef** [comment]
Yes, Electron ended [Windows 7 and 8.1](https://www.electronjs.org/blog/windows-7-to-8-1-deprecation-notice) support in version 23, and we recently updated to 25.3.0 so Windows 7 and 8.1 are no longer supported.

**Comment by KeeJef** [comment]
Updated .readme here https://github.com/oxen-io/session-desktop/pull/2899

**Comment by blinkenlight** [comment]
So why the hell are you then not detecting an "unsupported" (do you seriously think I'll move away from W7 for this...?) OS with the installer and abort the upgrade with a message that explains we're being dumped...? This is NOT a closed issue.
